code/modules/events/spider_terror.dm

```diff
@@ -22,23 +22,31 @@
 			if(temp_vent.parent.other_atmosmch.len > 50)
 				vents += temp_vent
 	var/spider_type
-	var/infestation_type = pick(1, 2, 3, 4, 5)
+	var/highpop_trigger = 80
```

Comment:
best to make this a local define. Be sure to undef it at the end of the file

Reply:
Why? That would make it 3 lines rather than 1, for a var that's only used once.

Reply:
A define is usually something that you first notice when opening the file since it's at the top. It also stands out. Performance wise it's a tiny improvement but nothing to write home about.
It is mostly for consistency with how it's done in other parts of the codebase. Variables can be used if they can be changeable through inheritance or admemery and such. (only works if they are defined in the object though. Not inside a proc)